Sources/Sentry/SentryTransaction.m

```diff
@@ -87,4 +66,65 @@ - (void)finish
     [self.hub captureTransaction:self];
 }
 
+- (SentrySpanId *)spanId
+{
+    return self.spanContext.spanId;
+}
+
+- (SentryId *)traceId
+{
+    return self.spanContext.traceId;
+}
+
+- (BOOL)isSampled
+{
+    return self.spanContext.sampled;
+}
+
+- (NSString *)spanDescription
+{
+    return self.spanContext.spanDescription;
+}
+
+- (void)setSpanDescription:(NSString *)spanDescription
+{
+    [_spanContext setSpanDescription:spanDescription];
+}
+
+- (SentrySpanStatus)status
+{
+    return self.spanContext.status;
+}
+
+- (void)setStatus:(SentrySpanStatus)status
+{
+    [self.spanContext setStatus:status];
+}
+
+- (NSString *)operation
+{
+    return self.spanContext.operation;
+}
+
+- (void)setOperation:(NSString *)operation
+{
+    [self.spanContext setOperation:operation];
+}
+
+- (NSDictionary<NSString *, id> *)serialize
+{
+    NSMutableDictionary<NSString *, id> *serializedData =
+        [[NSMutableDictionary alloc] initWithDictionary:[super serialize]];
+    serializedData[@"spans"] = @[];
```

Comment:
`l`: I guess this is still empty because we still need to add spans? If so, please add a comment, so I don't wonder why.

Reply:
Exactly. Spans coming soon.